bin/setup-jsdom-version.sh

```diff
@@ -8,10 +22,20 @@ cmds_exist() {
     fi
 }
 
+
 if [ cmds_exist ]; then
     NODE_VERSION=$(node -v);
 
     if [[ $NODE_VERSION == v0.1* ]]; then
-        npm install jsdom@3 --save
```

Comment:
Is this one line change the same as what you did?
 `npm list jsdon@3 || npm install jsdom@3 --save`